third_party/terraform/tests/resource_container_node_pool_test.go.erb

```diff
@@ -238,7 +238,85 @@ func TestAccContainerNodePool_withSandboxConfig(t *testing.T) {
 		},
 	})
 }
+<% end -%>
+
+<% unless version.nil? || version == 'ga' -%>
+func TestAccContainerNodePool_withKubeletConfig(t *testing.T) {
+	t.Parallel()
+
+	cluster := fmt.Sprintf("tf-test-cluster-%s", randString(t, 10))
+	np := fmt.Sprintf("tf-test-np-%s", randString(t, 10))
+
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckContainerClusterDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccContainerNodePool_withKubeletConfig(cluster, np, "static"),
```

Comment:
Can we get an update test here too? I'd specifically like to see updating some of the fields to empty to check whether or not we need to ForceSendFields them.

Reply:
I have a test to support changing the timeout value, but `cpu_manager_policy` (the more interesting one) can't be tested because the API rejects the second value right now.